components/map/basic/src/leaflet/marker-manager.js

```diff
@@ -141,8 +141,12 @@ class MarkerManager {
   }
 
   // Coupled FC, should be removed in the future
-  isVisited() {
-    return false
+  isVisited({propertyInfo}) {
```

Comment:
Quizá se puede reducir a:

```suggestion
  isVisited({propertyInfo}) => !!propertyInfo && !!propertyInfo.isVisited
```